Gems/GeoJSONSpawner/Code/Source/GeoJSONSpawner/GeoJSONSpawnerUtils.h

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (C) Robotec AI - All Rights Reserved
+ *
+ * This source code is protected under international copyright law.  All rights
+ * reserved and protected by the copyright holders.
+ * This file is confidential and only available to authorized individuals with the
+ * permission of the copyright holders.  If you encounter this file and do not have
+ * permission, please contact the copyright holders and delete this file.
+ */
+
+#pragma once
+
+#include "GeoJSONSpawner/GeoJSONSpawnerTypeIds.h"
+
+#include <AzCore/Math/Vector3.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <AzFramework/Spawnable/Spawnable.h>
+#include <AzFramework/Spawnable/SpawnableEntitiesInterface.h>
+
+namespace GeoJSONSpawner::GeoJSONUtils
+{
+    using Coordinates = AZStd::vector<AZStd::array<double, 3>>;
+    using SpawnableCoordinatesMap = AZStd::unordered_map<AZStd::string, Coordinates>;
+    using DespawnCallback = AZStd::function<void(AzFramework::EntitySpawnTicket::Id)>;
+    using Ids = AZStd::unordered_set<int>;
+
+    enum class GeometryType
+    {
+        Point = 0,
+        MultiPoint,
+        LineString,
+        MultiLineString,
+        Polygon,
+        MultiPolygon,
+        GeometryCollection,
+        Unknown
+    };
+
+    //! Information about a location to spawn an entity.
```

Comment:
Do you mean that this class holds the information about features from geojson? On the other hand, shouldn't we call the "features" instead of "objects"?

From geojson.org:
> Geometric objects with additional properties are Feature objects. Sets of features are contained by FeatureCollection objects.

```suggestion
    //! Information about a location of GeoJSON feature.
```